Documentation/ReleaseNotes.md

```diff
@@ -102,6 +102,15 @@ MRTK has added initial support for [Unity 2019.3's new XR platform](https://blog
 
 Please see [Known issues](#known-issues-in-230) for details on known limitations.
 
+**Assets/Dependencies folder**
+
+After MSBuild for Unity is enabled, a Dependencies folder will be created in the project. This folder contains the plugins (ex: DotNetWinRT) that are imported, by MRTK.
```

Comment:
Can this folder be ignored in a git repo, or does it need to be included?

Reply:
good question. MSBuild for Unity should restore the folder. though some customers may want to checkin the folder

Reply:
i can add that this is an optional folder for source control